### PR TITLE
Yolk issue fix

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

Uhhh, I made it so yolks give 5u instead of 4 when cracked into a beaker.  Pretty much that

## Why It's Good For The Game

It's weird to not get enough yolk to make a cake when cracked into a beaker. Having to crack a 4th egg because you did it in the wrong container kinda sucks.

## Changelog
:cl:
fix: Eggs now give 5u yolk no matter what it is cracked in
/:cl:

### CODE REVIEW

**Comment by RaveRadbury** [comment]
You might need to remove the bit of code that made an exception for drinking glasses.

**Comment by carshalash** [comment]
That kinda just, exists to make it so it can be cracked into the glass.

File: code/modules/food_and_drinks/recipes/food_mixtures.dm [hunk 1]
```diff
@@ -157,7 +157,7 @@
 
 /datum/chemical_reaction/food/pancakebatter
 	results = list(/datum/reagent/consumable/pancakebatter = 15)
-	required_reagents = list(/datum/reagent/consumable/eggyolk = 12, /datum/reagent/consumable/milk = 10, /datum/reagent/consumable/flour = 5)
+	required_reagents = list(/datum/reagent/consumable/eggyolk = 15, /datum/reagent/consumable/milk = 10, /datum/reagent/consumable/flour = 5)
```
Comment (LemonInTheDark) [inline code/modules/food_and_drinks/recipes/food_mixtures.dm:160]: I mean it's minor as fuck, but why?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/recipes/food_mixtures.dm (hunk 1 above)
Comment (carshalash) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm:160]: It was weird having it be the only eggception to the 5u egg thing. Everything is kind of uniform now.

Comment (LemonInTheDark) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm:160]: aight let's jive

**Pull request merged**